lib/exchange.dart

```diff
@@ -69,6 +72,29 @@ class _ExchangeWidgetState extends State<ExchangeWidget> {
     widget.websocket.wsEvent.unsubscribe(_websocketEvent);
   }
 
+  Widget _amountSelected(String amountName, void Function() onPressed) {
```

Comment:
this should take a `AmountSliderSelected` parameter not string, then it can also imply the `onPressed` parameter

better just one function instead of two, something like: `_createSliderButton(AmountSliderSelected amount, bool selected)`